lib/internal/process/next_tick.js

```diff
@@ -88,20 +109,58 @@ function setupNextTick() {
     }
   }
 
+  // TODO(trevnorris): Using std::stack is much slower here than was the
+  // Float64Array stack used in a previous implementation. Problem is the
+  // Float64Array stack was a bit brittle. Investigate how to harden that
+  // implementation and possibly reintroduce it.
```

Comment:
Ah, thanks for explaining! :+1: Maybe the comment could mention `ids_stack_` explicitly, though ;)